packages/peregrine/lib/talons/CartPage/PriceAdjustments/useCouponCode.js

```diff
@@ -87,26 +91,28 @@ export const useCouponCode = props => {
         setIsCartUpdating
     ]);
 
-    let applyErrorMessage;
+    let derivedErrorMessage;
+
+    if (applyError || removeCouponError) {
+        const errorTarget = applyError || removeCouponError;
 
-    if (applyError) {
-        if (applyError.graphQLErrors) {
+        if (errorTarget.graphQLErrors) {
```

Comment:
OOPS - as we just discovered, `graphQLErrors` is an array. All these checks will need to be updated to `length`

Reply:
Great catch! Added this check to `FormError`, and we'll address the other areas once we create a utility.